pkg/watcher/watcher.go

```diff
@@ -162,13 +162,13 @@ func (watcher *K8sWatcher) FindPod(podID string) (*corev1.Pod, error) {
 	}
 
 	allPods := watcher.podInformer.GetStore().List()
-	if pod, ok := findPod(podID, allPods); ok {
+	if pod, ok := findPod(allPods); ok {
 		return pod, nil
 	}
 	return nil, fmt.Errorf("unable to find pod with ID %s (index pods=%d all pods=%d)", podID, len(objs), len(allPods))
 }
 
-func findPod(podID string, pods []interface{}) (*corev1.Pod, bool) {
+func findPod(pods []interface{}) (*corev1.Pod, bool) {
```

Comment:
I guess it was switched to the pod's structure to retrieve the ID thing but this could be unexpected